x-pack/plugins/serverless_search/public/application/components/connectors/self_managed_connectors_empty_prompt.tsx

```diff
@@ -0,0 +1,255 @@
+                        </EuiFlexItem>
+                        <EuiFlexItem>
+                          <EuiIcon size="m" type="sortRight" />
+                        </EuiFlexItem>
+                        <EuiFlexItem>
+                          <EuiIcon color="primary" size="l" type={connectorsIcon} />
+                        </EuiFlexItem>
+                        <EuiFlexItem>
+                          <EuiIcon size="m" type="sortRight" />
+                        </EuiFlexItem>
+                        <EuiFlexItem>
+                          <EuiIcon color="primary" size="l" type="logoElastic" />
+                        </EuiFlexItem>
+                      </EuiFlexGroup>
+                    </EuiFlexItem>
+                    <EuiFlexItem>
+                      <EuiText>
+                        <p>
+                          {i18n.translate(
+                            'xpack.serverlessSearch.connectorsEmpty.guideThreeDescription',
+                            {
+                              defaultMessage:
+                                'Enter access and connection details for your data source and run your first sync',
+                            }
+                          )}
+                        </p>
+                      </EuiText>
+                    </EuiFlexItem>
+                  </EuiFlexGroup>
+                </EuiFlexItem>
+              </EuiFlexGroup>
+            </EuiPanel>
+          </EuiFlexItem>
+        </EuiFlexGroup>
+      }
+      actions={
+        <EuiFlexGroup direction="row" gutterSize="m">
+          <EuiFlexItem>
+            <EuiButton
+              data-test-subj="serverlessSearchEmptyConnectorsPromptCreateConnectorButton"
+              disabled={!data?.canManageConnectors}
+              fill
+              iconType="plusInCircle"
+              onClick={() => createConnector()}
+              isLoading={isLoading}
+            >
+              {i18n.translate('xpack.serverlessSearch.connectorsEmpty.selfManagedButton', {
+                defaultMessage: 'Self-managed connector',
+              })}
+            </EuiButton>
+          </EuiFlexItem>
+          <EuiFlexItem>
+            <EuiFlexGroup direction="column" gutterSize="s" alignItems="center">
+              <EuiFlexItem>
+                <EuiButton
+                  data-test-subj="serverlessSearchEmptyConnectorsPromptCreateElasticManagedConnectorButton"
+                  isLoading={isLoading}
+                  onClick={() =>
+                    navigateToUrl(`${BASE_CONNECTORS_PATH}/${ELASTIC_MANAGED_CONNECTOR_PATH}`)
+                  }
+                >
+                  {i18n.translate('xpack.serverlessSearch.connectorsEmpty.elasticManagedButton', {
+                    defaultMessage: 'Elastic managed connector',
+                  })}
+                </EuiButton>
+              </EuiFlexItem>
+              <EuiFlexItem grow={false}>
+                <EuiBadge color="accent">Coming soon</EuiBadge>
```

Comment:
This should be translated

Reply:
Using a const for that and translated. Thanks.